pages/common/grep.md

```diff
@@ -23,6 +23,16 @@
 
 `grep --{{context|before-context|after-context}}={{3}} "{{search_pattern}}" {{path/to/file}}`
 
+You can also use these 3 flags to add more context of your search
+
+`-A num` = print x num of lines after each match
+
+`-B num` = print x num of lines before each match
+
+`-C num` = print x num of lines before and after each match
+
+`grep -C 3 "{{search_pattern}}" {{path/to/file}}`
```

Comment:
This example is the same than above just use a short argument version.

Reply:
sorry if this is spamming your inbox 😅

Reply:
np. This is a very popular command and it's hard to cover its functionality in just 8 basic examples